.github/workflows/check-semver.yml

```
@@ -11,7 +11,7 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  TOOLCHAIN: nightly-2024-06-01
+  TOOLCHAIN: nightly-2024-10-18
```

```suggestion
  TOOLCHAIN: nightly-2024-11-19
```
We can bump this higher, confirmed with https://github.com/paritytech/polkadot-sdk/actions/runs/11930690709/job/33252075574?pr=5732

---

I was going off of the version the semver lib supports but yeah the newer nightly also works so will use that.